rm -rf tags.sh build && git reset --hard
patch -Np1 -i "${mypatd:-"../"}/supertux.patch"
chmod 755 tags.sh
mkdir build && cd build && cmake -DIS_SUPERTUX_RELEASE=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr ..
diff a/tags.sh b/tags.sh
--- a/tags.sh	1969-12-31 17:00:00.000000000 -0700
+++ b/tags.sh	1969-12-31 17:00:00.000000000 -0700
@@ -0,0 +1,2 @@
+#!/bin/bash
+git ls-remote -q --tags --refs | grep -Pio 'v(\d+(\.|_|-|$)){2}[\w\.-]*' | perl -pe 's|[[:punct:]]|.|g' | sort -V | tail -1
diff a/CMakeLists.txt b/CMakeLists.txt
index 85ab833..dacfdfa 100644
--- a/CMakeLists.txt	1969-12-31 17:00:00.000000000 -0700
+++ b/CMakeLists.txt	1969-12-31 17:00:00.000000000 -0700
@@ -301,7 +301,8 @@ if(EXISTS "${CMAKE_SOURCE_DIR}/.git")
   find_program(GIT_EXECUTABLE git)
   if(NOT GIT_EXECUTABLE EQUAL "GIT_EXECUTABLE-NOTFOUND")
     include(GetGitRevisionDescription)
-    git_describe(VERSION_STRING_GIT "--tags" "--match" "?[0-9]*.[0-9]*.[0-9]*")
+#    git_describe(VERSION_STRING_GIT "--tags" "--match" "?[0-9]*.[0-9]*.[0-9]*")
+     execute_process(COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/tags.sh OUTPUT_VARIABLE VERSION_STRING_GIT)
     string(REPLACE "v" "" VERSION_LIST ${VERSION_STRING_GIT})
     string(REGEX REPLACE "(-|_|\\.)" ";" VERSION_LIST ";${VERSION_LIST}")
   endif()
